packages/components/tests/dummy/app/templates/components/button-set.hbs

```diff
@@ -0,0 +1,77 @@
+{{page-title "ButtonSet Component"}}
+
+<h2 class="dummy-h2">ButtonSet</h2>
+
+<section>
+  <h3 class="dummy-h3" id="overview"><a href="#overview" class="dummy-link-section">§</a> Overview</h3>
+
+  <p class="dummy-paragraph">Provides consistent layout and spacing of buttons.</p>
+</section>
+
+<section>
+  <h3 class="dummy-h3" id="component-api"><a href="#component-api" class="dummy-link-section">§</a> Component API</h3>
+  <p class="dummy-paragraph" id="component-api-button-set">
+    Here is the API for the component:
+  </p>
+  <dl class="dummy-component-props" aria-labelledby="component-api-button-set">
+    <dt>...attributes</dt>
+    <dd>
+      <p><code class="dummy-code">...attributes</code> spreading is supported on this component.</p>
+    </dd>
+  </dl>
+</section>
+
+<section>
+  <h3 class="dummy-h3" id="how-to-use"><a href="#how-to-use" class="dummy-link-section">§</a> How to use</h3>
+
+  {{! prettier-ignore-start }}
+  <CodeBlock
+    @language="markup"
+    @code='
+      <Hds::ButtonSet>
+        <Hds::Button @text="Button 1" />
+        <Hds::Button @text="Button 2" />
+        <Hds::Button @text="Button 3" />
+      </Hds::ButtonSet>
```

Comment:
What do you think of having a more realistic use case here? Maybe just a primary and a secondary button?

Reply:
Yes, that's what I was thinking would be good. Is there a good example of a more realistic typical usage? I can look around.

Reply:
The Figma docs:
<img width="753" alt="screenshot_1714" src="https://user-images.githubusercontent.com/686239/181359420-0dc4a940-7c98-43fc-981d-deca60a85bf1.png">

One primary button, next to a secondary button.